run_sharded_tests.sh

```
@@ -0,0 +1,205 @@
+                    wait $pid || failed=1
+                done
+                if [ $failed -ne 0 ]; then
+                    echo -e "${RED}✗ Parallel batch failed${NC}"
+                    exit 1
+                fi
+                echo -e "${GREEN}✓ Batch completed${NC}\n"
+            fi
+        done
+        ;;
+
+    all-at-once)
```

Shellcheck (0.11.0)

[error] Couldn't parse this case item. Fix to allow more checks.

(SC1073)